src/crypto/password/pbkdf2.clj

```diff
@@ -9,6 +9,13 @@
            javax.crypto.spec.PBEKeySpec
            org.apache.commons.codec.binary.Base64))
 
+(def algorithm-codes
+  {"HMAC-SHA1" "PBKDF2WithHmacSHA1"
+   "HMAC-SHA256" "PBKDF2WithHmacSHA256"})
+
+(defn- code-to-algorithm [code]
```

Comment:
This function isn't necessary, since it's functionally equivalent to `algorithm-codes`.